lib/recurly/client.rb

```diff
@@ -76,8 +79,16 @@ def get(path, **options)
       path = scope_by_site(path, **options)
       request = Net::HTTP::Get.new path
       set_headers(request, options[:headers])
-      http_response = run_request(request, options)
-      handle_response! request, http_response
+
+      retries = 0
+      begin
+        http_response = run_request(request, options)
+        handle_response! request, http_response
+      rescue Recurly::Errors::UnavailableError, Recurly::Errors::InternalServerError
+        retries += 1
+        retry if retries < 2 # Retry 5xx once for GET requests only
```

Comment:
Should we add sleeping before retrying?

Reply:
I think we should add some sleeps for certain network conditions, although it will pretty much be a guess :) I'm less concerned about sleeping in between server 5xx responses since the request is likely to route to a different backend process that may not have the same issue.